composer.json

```diff
@@ -22,15 +22,15 @@
     ],
     "require": {
         "php": "^7.2",
-        "friendsofsymfony/http-cache": "^2.5.2",
+        "friendsofsymfony/http-cache": "^2.6",
         "symfony/framework-bundle": "^3.4.26 || ^4.2.7 || ^5.0",
         "symfony/http-foundation": "^3.4.26 || ^4.2.7 || ^5.0",
         "symfony/http-kernel": "^3.4.26 || ^4.2.7 || ^5.0"
     },
     "require-dev": {
         "php-http/guzzle6-adapter": "^1.0 || ^2.0",
         "php-http/message": "^1.0 || ^2.0",
-        "mockery/mockery": "^1.0 || ^2.0",
+        "mockery/mockery": "^1.0",
```

Comment:
why this change? are things broken with mockery 2 in master?

Reply:
@dbu There exist no mockery 2: https://packagist.org/packages/mockery/mockery

Reply:
oh wow. i added that 2 years ago when allowing httplug 2 - seems i was over-motivated adding version 2 to things :man_facepalming: 

thanks for spotting!